README.md

```diff
@@ -42,7 +42,7 @@ Installation steps:
     * run `rake db:migrate`
     * OR
     * in MySQL, `drop database XXX;` for each database in `config/database.yml` and then try `rake db:setup` again
-* `rake db:seed` to populate it with initial data
+* `rake db:seed` to populate it with initial dummy data
```

Comment:
woops we should bring the AGPL discussion back to the group. This is just in the diffs.